_includes/homepage/infocols.html

```diff
@@ -0,0 +1,79 @@
+{%- comment -%}
+  include: 
+    subtitle // optional
+    title
+    url // optional
+    linktext // optional
+    infoboxes[]:
+      title
+      description // optional
+{%- endcomment -%}
+<section class="px-8 px-sm-16 px-md-24 py-24 {{ gray_container_class }}">
```

Comment:
could i get some clarity on what `gray_container_class` is and where it comes from?